lib/constant.ml

```diff
@@ -0,0 +1,10 @@
+let sexp_of_int = Base.sexp_of_int
+let sexp_of_float = Base.sexp_of_float
+let sexp_of_string = Base.sexp_of_string
+let sexp_of_list = Base.sexp_of_list
+let sexp_of_option = Base.sexp_of_option
```

Comment:
Perhaps `sexp_of_float`, `sexp_of_list`, and `sexp_of_option` are unnecessary?

Reply:
I see.  Thanks!